src/mongocxx/test/spec/client_side_encryption.cpp

```diff
@@ -267,12 +327,14 @@ TEST_CASE("Client side encryption spec automated tests", "[client_side_encryptio
 
     std::string test_file;
     while (std::getline(test_files, test_file)) {
-        if (std::find(unsupported_tests.begin(), unsupported_tests.end(), test_file) !=
-            unsupported_tests.end()) {
-            WARN("skipping " << test_file << " due to unsupported operation");
-            continue;
+        SECTION(test_file) {
```

Comment:
As @bazile-clyde taught me, wrapping this in a `SECTION` makes it much easier to run individual tests. For example, to run just `bypassAutoEncryption.json`:

```
./cmake-build/src/mongocxx/test/test_client_side_encryption_specs --section "bypassAutoEncryption.json"
```